api/common.js

```diff
@@ -19,6 +19,8 @@ const config = require('./config');
 const db = require('./models');
 const mongoose = require('mongoose');
 
+stopwords.add('undefined');
```

Comment:
Should I check for the variables which are undefined or use a stopword?

Reply:
What do you mean by "undefined"? How could it be undefined?

Reply:
I will put a check for readme instead!